src/toil/wdl/wdltoil.py

```diff
@@ -3514,6 +3556,11 @@ def main() -> None:
         # TODO: Move cwltoil's generate_default_job_store where we can use it
         options.jobStore = os.path.join(mkdtemp(), 'tree')
 
+    # Take care of incompatible arguments related to file imports
+    if options.run_imports_on_workers is True and options.import_workers_disk is None:
+        logger.error("Commandline arguments --runImportsOnWorkers and --importWorkersDisk must both be set to run file imports on workers.")
+        return
```

Comment:
We probably need to raise an error or exit with a nonzero exit code, or the workflow will look like it succeeded.

Reply:
Changed to a RuntimeError